test/gtest/uct/test_flush.cc

```diff
@@ -525,13 +498,8 @@ UCS_TEST_P(uct_flush_test, am_flush_ep_nb) {
     m_flush_flags = UCT_FLUSH_FLAG_LOCAL;
     if (is_caps_supported(UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE)) {
         test_flush_am_disconnect(&uct_flush_test::flush_ep_nb, false);
-
-        am_rx_count   = 0;
+        am_rx_count    = 0;
         m_flush_flags |= UCT_FLUSH_FLAG_CANCEL;
-        test_flush_am_disconnect(&uct_flush_test::flush_ep_nb, false);
```

Comment:
maybe test_flush_am_pending() should assert that if flags have CANCEL, destroy_ep parameter must be true?

Reply:
good point, I think you meant `test_flush_am_disconnect`, we have several flush functions in the test